package.json

```diff
@@ -60,6 +60,8 @@
         "refactor",
         "perf",
         "test",
+        "build",
+        "ci",
```

Comment:
This is the configuration for `validate-commit-msg` which uses itself to validate commit messages. This wont have any impact on consumers of the library.